libs/sdk-ui-dashboard/.dependency-cruiser.js

```diff
@@ -36,6 +36,8 @@ options = {
                 "src/presentation/scheduledEmail",
                 "src/presentation/shareDialog",
                 "src/presentation/saveAs",
+                "src/presentation/deleteDialog",
+                "src/presentation/saveAs",
```

Comment:
`src/presentation/saveAs` is mentioned twice now, please remove one.

Reply:
fixed